lib/manageiq/api/common/user.rb

```diff
@@ -0,0 +1,87 @@
+module ManageIQ
+  module API
+    module Common
+      class User
+        IDENTITY_KEY = 'x-rh-identity'.freeze
+
+        def self.current
+          Thread.current[:attr_current_user]
```

Comment:
@syncrou Should we raise an error here if the TLS is not set?

Reply:
I'll get that change in